tests/regression/rhbz-1708991/task.yaml

```diff
@@ -0,0 +1,32 @@
+summary: Check that snapd SELinux policy does not break systemd services with private mount ns
+
+systems: [fedora-*, centos-*]
+
+prepare: |
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB"/pkgdb.sh
+
+    if [[ "$SPREAD_SYSTEM" == centos-7-* ]]; then
+        distro_install_package systemd-resolved
+    fi
+
+    getenforce > enforcing.mode
+
+    # Enable enforcing mode, our policy is already marked as permissive, so we
+    # will get audit entries but the program will not be stopped by SELinux
+    setenforce 1
+    ausearch --checkpoint stamp -m AVC || true
+
+restore: |
+    setenforce "$(cat enforcing.mode)"
+    rm -f stamp enforcing.mode
+
+execute: |
+    # prepare does systemctl daemon-reexec to workaround:
```

Comment:
I don't see any `systemctl daemon-reexec` in this test. Maybe rephrase the comment to something like "the global prepare does ..." and also outline why this comment is relevant for this test?